gremlin-groovy-test/src/main/java/org/apache/tinkerpop/gremlin/groovy/jsr223/GroovyTranslatorTest.java

```diff
@@ -141,4 +144,52 @@ public void shouldHandleEmptyMaps() {
     public void shouldHaveValidToString() {
         assertEquals("translator[h:gremlin-groovy]", GroovyTranslator.of("h").toString());
     }
+
+    @Test
+    @LoadGraphWith(LoadGraphWith.GraphData.MODERN)
+    public void shouldHandleTraversalAddV() {
```

Comment:
the names of our tests are meant to convey information about what is being tested (when possible). to that end, you're not really testing "addV()" here but you're testing "string escaping", so something more like `shouldEscapeStrings()` would probably be a better name. I would probably combine this test with `shouldHandleVertex()` as they are both validating the "escaping" but if you have a better name for that test as it test something more specific than that then feel free to just rename the other test and keep them separate.

Reply:
I have renamed the test method and separate into two:
1. `shouldEscapeStrings`
2. `shouldHandleVertexAndEdge`